charts/linkerd2/values.yaml

```
@@ -298,6 +298,33 @@ prometheus:
   # - name: recording-rules
   #   subPath: recording_rules.yml
   #   configMap: linkerd-prometheus-rules
+  ###
+  ### Sidecar containers allow access to the prometheus data directory,
+  ### e.g. for exporting data to non-prometheus systems.
+  # sidecarContainers:
+  # - name: sidecar
+  #   image: gcr.io/myproject/stackdriver-prometheus-sidecar
+  #   lifecycle:
+  #     # n.b. only valid on k8s >=1.19
+  #     type: Sidecar
+  #   imagePullPolicy: always
```

```suggestion
  #   imagePullPolicy: Always
```

This configuration actually failed, so thought it would be better to have something that works atleast as a example :)